.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
looks like there is a single space before this bracket

Reply:
I don't see any space in the code I guess that +  is there to notify newly added line